themes/bootstrap5/templates/comments/userlist.phtml

```diff
@@ -0,0 +1,104 @@
+      <li class="nav-item" role="presentation" id="user-content-<?=$tab?>" data-tab="<?=$this->escapeHtmlAttr($tab)?>"><a class="<?='comments' === $tab ? 'active' : ''?> nav-link" href="<?=$this->url($tab . '-' . 'userlist')?>"><?=$this->transEsc(ucfirst($tab))?></a></li>
+    <?php endforeach; ?>
+  </ul>
+  <h2><?=$this->transEsc('Your Comments')?></h2>
+  <nav class="search-header hidden-print">
+    <?php
+      $abs = $this->comments->getAbsoluteItemNumber($this->comments->getItemCountPerPage());
+      $end = min($abs, $this->comments->getTotalItemCount());
+      $transParams = [
+        '%%start%%' => $this->localizedNumber($this->comments->getAbsoluteItemNumber(1)),
+        '%%end%%' => $this->localizedNumber($end),
+        '%%total%%' => $this->localizedNumber($this->comments->getTotalItemCount()),
+      ];
+    ?>
+    <div class="search-stats">
+      <?=$this->translate('showing_items_of_html', $transParams); ?>
+    </div>
+    <?=$this->context($this)->renderInContext('myresearch/controls/sort.phtml', ['sortList' => $this->sortList]); ?>
+  </nav>
+  <?php if (count($this->comments) !== 0): ?>
+    <form action="<?=$this->url('comments-deletecomments')?>" name="updateForm" id="select_all_comments" method="post">
+      <input type="hidden" value="<?=$this->escapeHtmlAttr($this->auth()->getManager()->getCsrfHash())?>" name="csrf">
+      <label>
+        <div class="checkbox">
+          <input type="checkbox" name="selectAll" class="checkbox-select-all">
+          <?=$this->transEsc('select_all_on_page')?>
+        </div>
+      </label>
+      <?=
+        $this->component(
+            'confirm-button',
+            [
+                'buttonId' => 'cancelSelected',
+                'buttonName' => 'cancelSelected',
+                'buttonLabel' => 'delete_selected',
+                'header' => 'confirm_usercomment_delete_selected_text',
+                'confirmId' => 'confirm_cancel_selected_yes',
+                'cancelClass' => 'confirm_cancel_no',
+                'ignoreLightbox' => true,
+            ]
+        )
+      ?>
+      <table class="usercontent-table table-striped table table-responsive">
+        <thead>
+          <tr>
+            <th></th>
+            <th><?=$this->transEsc('Created')?></th>
+            <th><?=$this->transEsc('Title')?></th>
+            <th><?=$this->transEsc('Comment')?></th>
+          </tr>
+        </thead>
+        <tbody>
+          <?php foreach ($this->comments as $comment):?>
+            <tr>
+              <td class="checkbox-col">
+                <div class="checkbox-container result">
+                  <div class="checkbox">
+                    <label>
+                      <span class="sr-only"><?=$this->transEsc('Select');?></span>
+                      <input class="checkbox-select-item" type="checkbox" name="deleteSelectedcomment[]" value="<?=$this->escapeHtmlAttr(($comment['id'] ?? ''))?>" id="checkbox_comment_<?=$comment['id']?>">
```

Comment:
Inconsistent handling of `$comment['id']` here -- if we need null coalescing, maybe we should assign the value in a separate line so we don't have to do it repeatedly.